bindings/python/pyproject.toml

```
@@ -41,7 +41,7 @@ benchmark = [
   "boto3-stubs[essential]",
 ]
 docs = ["pdoc"]
-test = ["behave", "pytest", "python-dotenv"]
+test = ["behave", "pytest", "python-dotenv", "pytest-asyncio"]
```

```suggestion
test = ["pytest", "python-dotenv", "pytest-asyncio"]
```

`behave` is unused now?

---

Have been removed in https://github.com/apache/incubator-opendal/pull/3253

---

Yes, I removed it in #3253

---

Right, just need to resolve conflicts.

---

> Yes, I removed it in #3253

Oh, I got @messense's point. This PR needs a rebase or merge.